backend/dal/Migrations/20210209065005_v01.07.03.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using Pims.Dal.Helpers.Migrations;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v010703 : SeedMigration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
+            migrationBuilder.AddColumn<int>(
+                name: "propertyTypeId",
+                table: "Parcels",
+                nullable: false,
+                defaultValue: 0);
+
+            migrationBuilder.AddColumn<int>(
+                name: "propertyTypeId",
+                table: "Buildings",
+                nullable: false,
+                defaultValue: 0);
```

Comment:
Wonder if the default value should be 1 here.  Would that reduce the need for your additional script to set it to 1?